pandas/io/parsers.py

```diff
@@ -656,7 +658,13 @@ def _get_options_with_defaults(self, engine):
         options = {}
 
         for argname, default in compat.iteritems(_parser_defaults):
-            options[argname] = kwds.get(argname, default)
+            value = kwds.get(argname, default)
+
+            if argname == 'mangle_dupe_cols' and value != default:
+                raise ValueError('Setting mangle_dupe_cols=False is '
+                                 'not supported yet')
```

Comment:
@jreback : Any response / update on this issue?  I think this is the only thing blocking this PR at the moment IINM.